### ISSUE TITLE
Focus bug in va-radio component

### ISSUE THREAD

**Issue by emisparkles** [department-of-veterans-affairs/vets-design-system-documentation#5547]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When clicking the Edit button to view a form with a `va-radio` component, focus is set on the page body instead of the first focusable form element. This means that nothing on the page is highlighted, and Mac VoiceOver utility does not announce the user's location in an editable form.

The video shows that the first element returned from [this `getFocusableElements` function](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/forms-system/src/js/utilities/ui/index.js#L46-L81) is `va-radio-option`.

https://github.com/user-attachments/assets/823c0c7b-6203-4004-82ca-a0dc235f70a1

## What I expected to happen

Clicking the Edit button opens the form with the first interactive input element highlighted, and Mac VoiceOver announces the item appropriately.

The video shows the expected behavior after removing `va-radio-option` from the [list of focusable elements](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/forms-system/src/js/web-component-fields/webComponentList.js#L16) so that the `getFocusableElements` function returns the first input item.

https://github.com/user-attachments/assets/d3e20214-cbaf-4666-a932-a0d9e2ef9f1d

## Reproducing

- CSS Library version:
- Device: Macbook
- Browser: Chrome

**URL(s) where this bug occurs**

- staging.va.gov/profile/health-care-settings/scheduling-preferences (try user 998 from [this list](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-vaos.md))

**Steps to reproduce:**

Locally in `vets-website` with a screen reader enabled:
1. Update [this constant ](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/user/selectors.js#L19-L20)to return `true` to enable the scheduling preference page
2. Run `yarn mock-api --responses src/applications/personalization/profile/mocks/server.js`
3. Run `yarn watch --env entry=profile`
4. Open http://127.0.0.1:3001/profile/health-care-settings/scheduling-preferences in the browser
5. Scroll down to "Appointment preferences" and click Edit under "Do you want help scheduling your appointments?"
6. Observe screen reader announcements & active element highlighting

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [x] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

### Details

I am on the Authenticated Experience team.

Call stack:
- [`ProfileInformationEditView`](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/user/profile/vap-svc/components/ProfileInformationEditView.jsx#L357) calls `getFocusableElements` and sets focus on the first element returned
- [`getFocusableElements`](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/forms-system/src/js/utilities/ui/index.js#L46-L81) checks elements in the `webComponentsList`
- [`webComponentsList`](https://github.com/department-of-veterans-affairs/vets-website/blob/94b125534f851969413b23959aa8369ff6d17e18/src/platform/forms-system/src/js/web-component-fields/webComponentList.js#L16) returns `va-radio-option` as first focusable element

**Comment by caw310** [comment]
Triage in sprint 24

**Comment by jeana-adhoc** [comment]
Note: Trying to load the app to QA, and running into issues.

**Comment by emisparkles** [comment]
This product is currently unavailable in staging due to test user restrictions. However, I was able to load the app locally and confirm that this is fixed. Thanks @jeana-adhoc and team!

https://github.com/user-attachments/assets/2d1b4de5-b604-441c-84d0-d2d5c6049485

https://github.com/user-attachments/assets/449b2737-1933-4ae8-bda8-8b33bcd63560

**Comment by jeana-adhoc** [comment]
Thanks @emisparkles !!

This is resolved.
